pkg/porter/credentials.go

```diff
@@ -233,9 +245,9 @@ func (p *Porter) ShowCredential(ctx context.Context, opts CredentialShowOptions)
 	}
 
 	credSet := DisplayCredentialSet{
-		SchemaType:    "CredentialSet",
 		CredentialSet: cs,
 	}
+	credSet.SchemaType = "CredentialSet"
```

Comment:
I wonder if we can declaring a constant for each `SchemaType` s will help to consolidate the hard coded string into one place